.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 22.1.0
+    rev: 22.6.0
```

Comment:
This is tangent, hence needs to be in its own commit to the main branch.

Reply:
sorry about that, I keep getting a pre-commit error and was trying some fixes, forgot to undo that one.

Reply:
Fixed via #1382